regression/cbmc/field-sensitivity15/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main.c
+
+^VERIFICATION SUCCESSFUL$
+^EXIT=0$
+^SIGNAL=0$
+--
+--
+The test is a minimized version of code generated from s2n-quic using Kani.
```

Comment:
More tests are always good but why is this in a documentation PR?  I guess I have missed something...